bahmni-commons-api/src/main/java/org/bahmni/module/bahmnicommons/contract/patient/search/PatientSearchBuilder.java

```diff
@@ -16,6 +18,8 @@
 
```

Comment:
This class itself is marked as deprecated and not used anymore.

Reply:
@angshu removed the deprecated class